web/elm/src/Dashboard/Models.elm

```diff
@@ -30,7 +31,7 @@ type alias Model =
             , isTeamsRequestFinished : Bool
             , isPipelinesRequestFinished : Bool
             , isResourcesRequestFinished : Bool
-            , isJobsErroring : Bool
+            , jobsError : Maybe FetchError
```

Comment:
It is probably worth making a similar change to the other `is<endpoint>Erroring` fields for consistency.

Reply:
did it